src/test/scala/dsptools/numbers/FixedPointSpec.scala

```diff
@@ -26,26 +28,109 @@ class FixedRing1(val width: Int, val binaryPoint: Int) extends Module {
 
 class FixedRing1Tester(c: FixedRing1) extends DspTester(c) {
   val increment: Double = if(c.binaryPoint == 0) 1.0 else 1.0 / (1 << c.binaryPoint)
-  println(s"Increment is $increment")
   for(i <- -2.0 to 3.0 by increment) {
-    println(s"Testing value $i")
-
     poke(c.io.in, i)
 
     expect(c.io.floor, breeze.numerics.floor(i), s"floor of $i should be ${breeze.numerics.floor(i)}")
     expect(c.io.ceil, breeze.numerics.ceil(i), s"ceil of $i should be ${breeze.numerics.ceil(i)}")
     expect(c.io.isWhole, breeze.numerics.floor(i) == i , s"isWhole of $i should be ${breeze.numerics.floor(i) == i}")
     expect(c.io.round, breeze.numerics.round(i), s"round of $i should be ${breeze.numerics.round(i)}")
```

Comment:
@shunshou I am not sure what you mean here, I am incrementing by the smallest increment possible for the binaryPoint under consideration